src/satosa/frontends/saml2.py

```diff
@@ -6,6 +6,7 @@
 import json
 import logging
 from urllib.parse import urlparse
+from distutils.util import strtobool
```

Comment:
strtobool should not be needed as the config is yaml and when loaded parses True/yes/true to boolean True and False/no/false to boolean False.